### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Department-stack**
nitpicking

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,9 +26,9 @@ sudo apt install libsqlite3-dev sqlite3 just cargo
 just build-release
 sudo just install
 ```
-_For Pop-os, write `COSMIC_DATA_CONTROL_ENABLED=1` in `/etc/environment`. (see [this issue](https://github.com/wiiznokes/clipboard-manager/issues/61))_
+Write `COSMIC_DATA_CONTROL_ENABLED=1` in `/etc/environment`. (see [this issue](https://github.com/wiiznokes/clipboard-manager/issues/61))_
```
Comment (wiiznokes) [inline README.md:29]: I think the italic will be broken without the first `_` ? Also, this workarround is not necessary for distro like fedora.

**Review by wiiznokes — CHANGES_REQUESTED** [review]
LGTM except the comment.

File: README.md (hunk 1 above)
Comment (Department-stack) [reply README.md:29]: Oh I see, also the workaround was necessary for me on fedora

Comment (wiiznokes) [reply README.md]: Which version ?

Comment (Department-stack) [reply README.md]: kde 40 lastest

Comment (wiiznokes) [reply README.md]: Oops, didn't see the response, thanks for letting me know. I hope to find a better solution soon

**Pull request merged**